test/e2e/deployment_appwrapper_test.go

```diff
@@ -45,7 +45,7 @@ func TestDeploymentAppWrapper(t *testing.T) {
 	defer func() {
 		_ = test.Client().Kueue().KueueV1beta1().ResourceFlavors().Delete(test.Ctx(), resourceFlavor.Name, metav1.DeleteOptions{})
 	}()
-	clusterQueue := createClusterQueue(test, resourceFlavor, 0)
+	clusterQueue := createClusterQueue(test, resourceFlavor, 0, "nvidia.com/gpu")
```

Comment:
You can leverage `Accelerator` for this, from https://github.com/opendatahub-io/distributed-workloads/blob/main/tests/kfto/support.go
Though the code would need to move to `codeflare-common` first.

Reply:
Please review - https://github.com/project-codeflare/codeflare-common/pull/94/files

Reply:
@sutaakar Could you please review again